source/common/http/conn_manager_impl.cc

```diff
@@ -278,7 +278,14 @@ void ConnectionManagerImpl::doDeferredStreamDestroy(ActiveStream& stream) {
 
   stream.completeRequest();
   stream.filter_manager_.onStreamComplete();
-  stream.filter_manager_.log();
+  if (codec_ && codec_->protocol() == Protocol::Http3 &&
+      !stream.filter_manager_.sawDownstreamReset() &&
+      Runtime::runtimeFeatureEnabled(
+          "envoy.reloadable_features.quic_defer_logging_to_ack_listener")) {
```

Comment:
this should be called out in release notes.

Reply:
done.